Tests/NostrSDKTests/RelayTests.swift

```diff
@@ -93,8 +104,10 @@ final class RelayTests: XCTestCase {
         let filter = try XCTUnwrap(Filter(kinds: [1], limit: 1))
         let subscriptionId = try relay.subscribe(with: filter)
         
-        wait(for: [receiveExpectation!], timeout: 10)
-        
+        wait(for: [receiveResponseEventExpectation!], timeout: 10)
```

Comment:
Since `wait(for:timeout:)` takes an array of expectations, you could just put them all into one wait.